### PR TITLE
Add last information and layout contest materials on home page.

### PR DESCRIPTION
**Pull request by verwoerd**

### CODE REVIEW

File: content/teams.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The following teams are competing in NWERC 2023. The numbers in parentheses repr
 | ![U-1097]  | Linköping University &mdash; SWE<ul><li>LiTHe Accepted (53)</li></ul>                                                                                                                                  |
 | ![U-1112]  | Lund University &mdash; SWE<ul><li>ECON 101 (37)</li><li>Noobs@LTH (106)</li><li>pi-pojkarna (143)</li></ul>                                                                                           |
 | ![U-1362]  | Norwegian University of Science and Technology &mdash; NOR<ul><li>Dataguttan (54)</li><li>NPC 2O23 (118)</li></ul>                                                                                     |
-| ![U-8002]  | Radboud University &mdash; NLD<ul><li>Quackamole (9)</li><li>Nope (69)</li><li>Bee and Puppycat (116)</li><li>It would have been wise to put a character limit on the team name field. (119)</li></ul> |
+| ![U-8002]  | Radboud University &mdash; NLD<ul><li>Quackamole (9)</li><li>Nope (69)</li><li>It would have been wise to put a character limit on the team name field. (119)</li><li>Bee and Puppycat (116)</li></ul> |
```
[the remark this answers is not in the document]
Comment (verwoerd) [reply content/teams.md:33]: crap, that is an automated double check for me to see if the teams page was still correct, and was not necessary to be comité 😓 . Will remove from commit.

File: content/news/last-information.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  university, team name, and team number.
+* During the contest, you are only allowed to use your Team Reference Document (TRD) as reference material. You may
+  bring three identical copies of the document per team, clearly marked with the university and team name. For
+  specifications on the TRD, please check the [rules page](/rules/). Your TRD should be handed in at registration. It is
+  not
+  possible to print the TRD at the contest site.
+* This year there will be no water bottle in the goody bag. We advise you to bring your own refillable water bottle. The
+  water bottles can be filled at the taps in the toilets. The water from Dutch water taps can be drunk safely.
+* People using the sleeping accommodations on campus can report at the [X building](https://map.tudelftcampus.nl/poi/x/)
+  on Friday night at 23:00 and Saturday at 22:00. It is still possible to register for sleeping spots [here](https://forms.gle/c9qByf4SNYbWKop99).
```
Comment (mpsijm) [inline content/news/last-information.md:25]: Is it a good idea to have this form posted online? :thinking:

Comment (verwoerd) [reply content/news/last-information.md:25]: it is already on https://nwerc.eu/accommodations/

**Review by mpsijm — APPROVED** [review]
I've simplified the comments! Last year, we used `<!--  -->` as well, a lot easier to hurriedly uncomment stuff when uploading the contest materials to the website :smile:

**Pull request merged**